online_archive/src/test/scala/FileUploaderSpec.scala

```diff
@@ -1,18 +1,25 @@
 import org.specs2.mock.Mockito
 import org.specs2.mutable.Specification
+import software.amazon.awssdk.core.async.AsyncRequestBody
 import software.amazon.awssdk.services.s3.S3Client
-import software.amazon.awssdk.services.s3.model.{HeadObjectRequest, HeadObjectResponse, ListObjectVersionsRequest, ListObjectVersionsResponse, NoSuchKeyException, ObjectVersion, PutObjectResponse, S3Exception}
-import software.amazon.awssdk.transfer.s3.{CompletedUpload, S3TransferManager, Upload, UploadRequest}
+import software.amazon.awssdk.services.s3.model._
```

Comment:
This should ideally be kept as it was unless we know only the objects originally imported exist.